src/sgame/sg_admin.cpp

```diff
@@ -529,6 +529,12 @@ void G_admin_cmdlist( gentity_t *ent )
 	char out[ MAX_STRING_CHARS ] = "";
 	unsigned len, outlen;
 
+	// If this is the local client, no need to send them commands.
```

Comment:
You could add a comment about how it's done in G_admin_register_cmds

LGTM

Reply:
Done.